signac/__main__.py

```diff
@@ -115,6 +116,9 @@
 """
 
 
+warnings.simplefilter("default")
```

Comment:
@glotzerlab/signac-maintainers Can I get a second opinion on this one-line fix for issue #317? I put it in this PR because it's related to deprecation warnings.

Reply:
I verified that warnings are sent to stderr, not stdout.